.teamcityTest/Gradle_Check_Tests/ApplyDefaultConfigurationTest.kt

```diff
@@ -14,6 +14,7 @@ import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.GradleBuildStep
 import model.CIBuildModel
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Disabled
```

Comment:
What happened here? Is this an unused import?

Reply:
It was a leftover. I removed locally and forgot to push.